src/main/target/KAKUTEF4/target.h

```diff
@@ -23,14 +23,16 @@
  */
 
 #pragma once
-#if defined(KAKUTEF4V2)
+#if defined(KAKUTEF4V2) || defined(KAKUTEF4V2TRI)
 #   define TARGET_BOARD_IDENTIFIER "KTV2"
 #   define USBD_PRODUCT_STRING "KakuteF4-V2"
 #else
 #   define TARGET_BOARD_IDENTIFIER "KTV1"
 #   define USBD_PRODUCT_STRING "KakuteF4-V1"
 #endif
 
+#define USE_TARGET_CONFIG
```

Comment:
Why this is here?

Reply:
Not sure why I added that, deleted......